app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is defined as a class variable, which means it is shared across all instances of `Animal`. This could lead to unintended behavior if multiple instances are created. Consider making `alive` an instance variable if you want each `Animal` instance to have its own list.